components/prod/childarrangement-familyinjunction.tf

```diff
@@ -0,0 +1,11 @@
+data "local_file" "childarrangement-familyinjunction-config" {
+  filename = "${path.cwd}/../../environments/prod/childarrangement-familyinjunction-service-gov-uk.yml"
+}
+
+module "childarrangement-familyinjunction" {
```

Comment:
Make sure you pass through the record types like in https://github.com/hmcts/azure-public-dns/blob/47cf4e0825c2f96e85461f1220641b77ff358f52/components/prod/platform-hmcts.tf#L7-L11

the divorce one I linked doesn't have it but better to get this right at the beginning

Reply:
addressed now.